lightkurve/targetpixelfile.py

```diff
@@ -831,6 +833,84 @@ def get_bkg_lightcurve(self, aperture_mask=None):
                           flux=np.nansum(self.flux_bkg[:, aperture_mask], axis=1),
                           flux_err=self.flux_bkg_err)
 
+
+    def _get_hdu(image, extension):
```

Comment:
This method appears inside the `KeplerTargetPixelFile` class, but it does not actually seem to operate on the class (e.g. it does not accept or use the `self` parameter).  This suggests it should be moved outside of the class to be a standalone function instead of a class method.

The only exception to the above rule is for so-called "static methods" which return a new instance of `KeplerTargetPixelFile` (like `from_archive`).